eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -219,6 +219,11 @@ steps:
     configuration: "$(BuildConfiguration)"
     msbuildArguments: "/MaxCPUCount /ConsoleLoggerParameters:Verbosity=Minimal;Summary;ForceNoAlign /t:AfterBuild /p:SignPackages=true /bl:$(Build.StagingDirectory)\\binlog\\12.SignPackages.binlog"
 
+- task: NuGetToolInstaller@1
+  displayName: Use NuGet 6.x
+  inputs:
+    versionSpec: 6.x
```

Comment:
[NuGetToolInstaller](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/nuget?view=azure-devops) has a `checkLatest`. Should we use that, so we don't need to maintain this version as NuGet bumps major version numbers?

Reply:
Thanks! I noticed that option too. Not sure if we should do that as it says `Enabling this option could cause unexpected build breaks when a new version of NuGet is released`.

Reply:
If there's anyone who should be notified of when changes to nuget.exe cause breaks in customer build pipelines, it's us. Also, we should know how we use nuget.exe in our pipeline (other tasks that use `nuget.exe install` to download their binaries, but we don't restore our projects with nuget.exe), so we can gauge risk.